.github/workflows/buildcheck.yml

```diff
@@ -20,11 +20,10 @@ jobs:
           python -V
           python -m pip --quiet --no-input install --upgrade pip
           python -m pip --quiet --no-input install --upgrade  wheel
-          cd vss-tools
-          pip install -e .
           pip install pytest
           # idlc currently used both during python test and later in this script
           pip install cyclonedds
+          pip install git+https://github.com/COVESA/vss-tools@master
```

Comment:
Using `master` as reference could give us some problems. Like recently when we did quite large refactoring of vss-tools. If we would have referred to `master` like here then VSS build on master would fail as we would drag in the new incompatible CLI syntax. So far upgrading vss-tools version have been a deliberate action, and to update the submodule you also need to fix regressions, if any. So the question is - do we see it as OK that VSS builds sometimes fails because of changes in vss-tools (even if vss repo is unchanged)

With that said, I sort of like the idea to get rid of the submodule. But what we possibly could do instead would be to rely on pypi-releases, including pre-releases. Or at least do it as part of release preparations. I.e. when we prepare the VSS 5.0 Release Candidate we could instead of `master` use a reference to `5.0rc0` in pypi. Sometimes like that is anyway needed for maintenance/release-branches, as an older VSS version may not necessarily may be compatible with a newer vss-tools version.